column.go

```diff
@@ -92,11 +92,14 @@ func (c *Column) Column(name string) *Column {
 
 // Pages returns a reader exposing all pages in this column, across row groups.
 func (c *Column) Pages() Pages {
+	if c.file == nil {
+		return emptyPages{}
+	}
 	return c.PagesFrom(c.file.reader)
 }
 
 func (c *Column) PagesFrom(reader io.ReaderAt) Pages {
-	if c.index < 0 {
+	if c.index < 0 || c.file == nil || reader == nil {
```

Comment:
Can we drop the check for `reader == nil`? That seems like fertile ground for subtle bugs, I don't want these methods to allow receiving a `nil` reader, that should be an invalid use of the API and triggering a panic helps identify what went wrong.
```suggestion
	if c.index < 0 || c.file == nil {
```